src/ExtendedTableWidget.cpp

```diff
@@ -314,35 +322,42 @@ void ExtendedTableWidget::copy(const bool withHeaders)
     const QString fieldSepText = "\t";
     const QString rowSepText = "\r\n";
 
+    QString sqlInsertStatement = QString("INSERT INTO \"%1\" ( '").arg(m->currentTableName().toString());
```

Comment:
Can you use ```sqlb::escapeIdentifier()``` here for quoting the table name? This way we have one function which does the quoting which makes changing the quotes relatively easy (somewhere we have an issue for this).

Reply:
Done